### PR TITLE
planner: update some UTs from cost model1 to model2

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve?

Issue Number: ref #35240

Problem Summary: planner: update some UTs from cost model1 to model2

### What is changed and how it works?
planner: update some UTs from cost model1 to model2

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: executor/executor_issue_test.go [hunk 1]
```diff
@@ -596,7 +598,7 @@ func TestIssue30382(t *testing.T) {
 		"SelectLock 6400.00 root  for update 0",
 		"└─HashJoin 6400.00 root  CARTESIAN inner join, other cond:or(gt(Column#8, 1), or(ne(test.t1.c_str, Column#7), if(ne(Column#9, 0), NULL, 0)))",
 		"  ├─Selection(Build) 0.80 root  ne(Column#10, 0)",
-		"  │ └─StreamAgg 1.00 root  funcs:max(Column#17)->Column#7, funcs:count(distinct Column#18)->Column#8, funcs:sum(Column#19)->Column#9, funcs:count(1)->Column#10",
```
Comment (qw4990) [inline executor/executor_issue_test.go:599]: Expected, model2 prefers to use HashAgg.

File: planner/core/integration_test.go [hunk 2]
```diff
@@ -5702,46 +5704,46 @@ func TestIndexJoinCost(t *testing.T) {
 	tk.MustExec("set @@tidb_enable_chunk_rpc = on")
 
 	tk.MustQuery(`explain format=verbose select /*+ TIDB_INLJ(t_outer, t_inner_pk) */ * from t_outer, t_inner_pk where t_outer.a=t_inner_pk.a`).Check(testkit.Rows( // IndexJoin with inner TableScan
-		`IndexJoin_11 12487.50 206368.09 root  inner join, inner:TableReader_8, outer key:test.t_outer.a, inner key:test.t_inner_pk.a, equal cond:eq(test.t_outer.a, test.t_inner_pk.a)`,
```
Comment (qw4990) [inline planner/core/integration_test.go:5705]: Expected, just update cost values.

**Review by Reminiscent — APPROVED** [review]

**Review by fzzf678 — APPROVED** [review]

**Pull request merged**